pkg/rbac/cache_test.go

```diff
@@ -0,0 +1,22 @@
+package rbac
+
+import (
+	"testing"
+	"time"
+)
+
+func TestBasic(t *testing.T) {
+	utr := New()
+	time := time.Time(time.Now())
+	utr.SetTokenTime("sha256~hX8hqMUOjTk1lcL92DXLZwZQvqmIwUPs56ehHzY-qVM", time)
```

Comment:
I'd make these look less real to avoid getting flagged by secure engineering tools.
You could change to something like this.
```
sha256~thisIsAMockTokenUsedForTesting
```